src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  extraClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  displayClass: string;
+  textColorClass: string;
+  extraClasses?: string;
+}
+
+export const UserFeedbackState = {
+  VALID: 'VALID',
+  WARNING: 'WARNING',
+  ERROR: 'ERROR',
+};
+
+export const TextColorClasses = {
+  default: 'text-dark-grey',
+  error: 'text-red'
+};
+
+export class UserFeedback extends React.Component<IUserFeedbackProps, any> {
+  render() {
+    let style = this.adjustStyle(this.props.state);
+    return (
+      <div className={`${style.textColorClass} ${style.displayClass}${style.extraClasses}`}>
```

Comment:
It was not a mistake but I refactored these lines and some others to add one. I agree that it is less confusing. Thanks so much Germain.